ApplicationLibCode/GeoMech/GeoMechDataModel/RigFemPartResultsCollection.cpp

```diff
@@ -216,19 +215,12 @@ void RigFemPartResultsCollection::setActiveFormationNames( RigFormationNames* ac
 {
     m_activeFormationNamesData = activeFormationNames;
 
-    RimProject* project = RimProject::current();
-    if ( project )
+    RimWellLogPlotCollection* plotCollection = RimMainPlotCollection::current()->wellLogPlotCollection();
+    if ( plotCollection )
```

Comment:
No need to check the plotCollection ptr?

Reply:
no, it is always there